projects/core/src/user/store/user-state.ts

```diff
@@ -11,6 +11,8 @@ import {
 } from '../../occ/occ-models/index';
 
 export const USER_FEATURE = 'user';
+export const USER_UPDATE_PROCESS = 'updateUserDetails';
```

Comment:
Could we match the the actions nomenclature and call the process something like `PROCESS_UPDATE_USER_DETAILS`.  I think it would improve the readability of the code that uses the constant.
Lets discuss.

Reply:
Done